### DISCUSSION TITLE
Permissions disappear and appear

### DISCUSSION THREAD

**Discussion by casudinlix** [spatie/laravel-permission#2359] [Q&A] [answered]
Hi everyone, I want to ask a question regarding the use of permissions in the javascript dataTable.
sample video [Video](https://jam.dev/c/2fe824dc-6464-4625-9c04-7d958596242d)
here is the code i have : 
![image](https://user-images.githubusercontent.com/15974968/225232686-8b17d8b2-4e06-4daa-83fc-2f241f1b47ae.png)

what do you think the cause is?
thank you

**Comment by parallels999** [comment]
Very little information, make an example app demonstrating the problem

**Reply by casudinlix** [reply]
hi, an example of the problem is in the video that I embedded above.
thank you

**Reply by parallels999** [reply, marked as answer]
Video is not an example, i can't run it on my environment, your video don't give me laravel version, php version, permission version, configs and others, please be profesional if you wan't a solution,
Imagine that you want your vehicle to be fixed and you just send a driving video to the mechanic and ask him what it has

Follow this: [Demo App Example](https://spatie.be/docs/laravel-permission/v5/basic-usage/new-app)
